### PR TITLE
update grpc-go dependency

### PR DESCRIPTION
**Pull request by bergwolf**
The connection closing errors are treated as temporary errors and grpc listener will just sleep a bit and retry again.

https://github.com/grpc/grpc-go/blob/5fe10fccafd3e35e84837ecc5a66e0f9600ab64e/internal/transport/transport.go#L659

That is fixed in the latest grpc code where the serve() function waits on a channel that is closed by server.Stop().

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

File: grpc.go [hunk 1]
```diff
@@ -756,7 +756,7 @@ func (a *agentGRPC) SignalProcess(ctx context.Context, req *pb.SignalProcessRequ
 
 	proc, err := ctr.getProcess(req.ExecId)
 	if err != nil {
-		return emptyResp, grpcStatus.Errorf(grpc.Code(err), "Could not signal process: %v", err)
+		return emptyResp, grpcStatus.Errorf(grpcStatus.Convert(err).Code, "Could not signal process: %v", err)
```
Comment (caoruidong) [inline grpc.go]: Code()

**Comment by jcvenegas** [comment]
@bergwolf could you fix the build errors:
```
go build -o kata-agent -ldflags "-X main.version=1.2.0-dcc2791078e9206eb82ddcddbd31d61701005912"
# github.com/kata-containers/agent
./grpc.go:33:2: imported and not used: "github.com/kata-containers/agent/vendor/google.golang.org/grpc"
./grpc.go:759:62: cannot use "github.com/kata-containers/agent/vendor/google.golang.org/grpc/status".Convert(err).Code (type func() codes.Code) as type codes.Code in argument to "github.com/kata-containers/agent/vendor/google.golang.org/grpc/status".Errorf
make: *** [kata-agent] Error 2
```

**Comment by bergwolf** [comment]
CI green. @kata-containers/agent PTAL.

**Comment by caoruidong** [comment]
lgtm

**Comment by bergwolf** [comment]
travis is stuck and doesn't even track this PR (https://travis-ci.org/kata-containers/agent/pull_requests). Let's ignore it as all the other CIs have passed.

`continuous-integration/travis-ci Expected — Waiting for status to be reported`

**Pull request merged**